pkg/pocketsizefund/config/config.py

```diff
@@ -3,32 +3,27 @@
 import argparse
 import datetime
 
-import toml
-
 ENVIRONMENT_DEVELOPMENT = "development"
 TIMEZONE = datetime.timezone.utc  # noqa: UP017
 
 
-class SAMConfig:
-    """SAMConfig class."""
+class Config:
+    """Config class."""
 
-    def __init__(self, file_path: str, environment: str = ENVIRONMENT_DEVELOPMENT) -> None:
-        """SAM Config class.
+    def __init__(self, environment: str = ENVIRONMENT_DEVELOPMENT) -> None:
+        """Config class.
 
         Args:
-            file_path (str): Path to SAM config file.
             environment (str, optional): Environment to use. Defaults to ENVIRONMENT_DEVELOPMENT.
         """
-        self.samconfig_file = toml.load(file_path)
-
-        self.parameters: dict[str, str] = {}
-        parameters = self.samconfig_file[environment]["deploy"]["parameters"]
-        for parameter in parameters["parameter_overrides"]:
-            parameter_split = parameter.split("=")
-            self.parameters[parameter_split[0]] = parameter_split[1]
+        self.parameters = {}
+        with open(f"etc/.env.{environment.lower()}") as config_file:  # noqa: PTH123
+            for line in config_file:
+                key, value = line.strip().split("=", 1)
+                self.parameters[key.strip()] = value.strip()
```

Comment:
right now we have most of our environment in the cluster secrets and are passing them into services directly, do we want to just lean into that?

or maybe there is a case for having a config module for things not in that config, but then we wouldn't need `etc/.env.{{ environment }}`

Reply:
Ah I think you're right. Given we can just spin up the cluster wherever that'll be sufficient. Should we just deprecate any file reading logic but retain stuff like the timezone, @chrisaddy ?